apertium-uzb-kaa.uzb-kaa.dix

```diff
@@ -19553,6 +19553,4846 @@
+<e>       <p><l>Astafyev<s n="np"/><s n="cog"/></l>    <r>Astafyev<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Astafyeva<s n="np"/><s n="cog"/></l>    <r>Astafyeva<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Astahov<s n="np"/><s n="cog"/></l>    <r>Astahov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Atanackovich<s n="np"/><s n="pat"/></l>    <r>Atanackovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Atanasiyevich<s n="np"/><s n="pat"/></l>    <r>Atanasiyevich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Atoyev<s n="np"/><s n="cog"/></l>    <r>Atoyev<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Atoyeva<s n="np"/><s n="cog"/></l>    <r>Atoyeva<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Atroshenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Atroshenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Auresku<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Auresku<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Avakov<s n="np"/><s n="cog"/></l>    <r>Avakov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Avasov<s n="np"/><s n="cog"/></l>    <r>Avasov<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Avdich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Avdich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Averchenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Averchenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Avramenko<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Avramenko<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Avramovich<s n="np"/><s n="pat"/></l>    <r>Avramovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Avramovskiy<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Avramovskiy<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Avtayev<s n="np"/><s n="cog"/></l>    <r>Avtayev<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Avtayeva<s n="np"/><s n="cog"/></l>    <r>Avtayeva<s n="np"/><s n="cog"/></r></p></e>
+<e>       <p><l>Ayanovich<s n="np"/><s n="pat"/></l>    <r>Ayanovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Aydin<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Aydin<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Aynachich<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Aynachich<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Ayup<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Ayup<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Ayupovich<s n="np"/><s n="pat"/></l>    <r>Ayupovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Ayvanovich<s n="np"/><s n="pat"/></l>    <r>Ayvanovich<s n="np"/><s n="pat"/></r></p></e>
+<e>       <p><l>Ayvazovskaya<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Ayvazovskaya<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
+<e>       <p><l>Ayvazovskiy<s n="np"/><s n="cog"/><s n="mf"/></l>    <r>Ayvazovskiy<s n="np"/><s n="cog"/><s n="mf"/></r></p></e>
```

Comment:
Some of these seem to be gendered?  If so, they shouldn't have `mf`, but the correct gender.